dev-libs/qxlsx/qxlsx-1.4.5.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multibuild cmake
+
+DESCRIPTION="Excel file(*.xlsx) reader/writer library using Qt"
+
+HOMEPAGE="https://github.com/QtExcel/QXlsx"
+SRC_URI="https://github.com/QtExcel/QXlsx/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+# soversion
+SLOT="0/0.1.4.4"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~riscv"
```

Comment:
any reason why ARCH=x86 dropped? would be great if you can document it in commit message explicitly

Reply:
Not intentional. It was a race: the old version gained the keyword after I opened this PR. Updated.